uidInList requires pidInList to be set (#553)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/Functions/Select.rst

   **Note:** :typoscript:`this` is a *special keyword* and replaced with the id of the
   *current record*.
   
   .. important:: 

>       :ref:`pidInList` has to be set or there will be no results. Setting

Reviewer: Maybe we should emphasize the default `this` for `pidInList` instead. There is actually no relation between these two options, both have their own unrelated behavior.

Author: Ok, well I never had this case before. Did you verify? Usually records are stored in sys_folders while the DataProcessor is run on pages. So I see almost no case where setting uidInList without pidInList with results

Author: Ok, so I added the default value to pidInList (as we didn't have it documented yet) and changed the important text accordingly

   **Note:** :typoscript:`this` is a *special keyword* and replaced with the id of the
   *current record*.
   

>    .. important:: 

Reviewer: ```suggestion
   .. attention:: 
```

As we know, we should avoid `important` . ;-)

>       :ref:`pidInList` is set to :typoscript:`this`. Therefore by defalault only records
>       from the current page are available for uidInList. Setting
>       :typoscript:`pidInList = 0` explicitely disables the check for the pid.

Reviewer: ```suggestion
      :ref:`pidInList` defaults :typoscript:`this`. Therefore by default only records
      from the current page are available for :typoscript:`uidInList`. If records
      should be fetched globally, :typoscript:`pidInList = 0` should also be set.
```
